src/components/modal/modal.js

```diff
@@ -427,15 +427,17 @@ export const BModal = /*#__PURE__*/ Vue.extend({
       }
     },
     computedModalAttrs() {
+      const { isVisible, ariaLabel } = this
+
       return {
         id: this.modalId,
-        role: 'dialog',
-        'aria-hidden': this.isVisible ? null : 'true',
-        'aria-modal': this.isVisible ? 'true' : null,
-        'aria-label': this.ariaLabel,
+        role: isVisible ? 'dialog' : null,
```

Comment:
I think `role="dialog"` can be left on, as `aria-hidden="true"` will hide that role from screen reader users until shown (when `aria-hidden` is removed)